cranelift/filetests/filetests/runtests/sadd_overflow.clif

```diff
@@ -0,0 +1,153 @@
+test interpret
+test run
+set enable_llvm_abi_extensions=true
+target x86_64
+target aarch64
+
+function %saddof_i128(i128, i128) -> i128 {
+block0(v0: i128,v1: i128):
+    v2, v3 = sadd_overflow v0, v1
+    return v2
```

Comment:
It might make the tests a little easier to read and maintain if we use multiple-returns here -- if we return `-> i128, i8` and compare against `[val, of]` we can collapse this with the below. See e.g. `runtests/i128-concat-split.clif` for an example.

Reply:
I changed that.